### ISSUE TITLE
Programs Not Installing

### ISSUE THREAD

**Issue by tkiral** [ChrisTitusTech/winutil#91]
**Describe the bug**
Programs do not install despite program indicating programs were installed.

**To Reproduce**
1. Start script
2. Choose "Install" from the left-hand navigation pane
3. Choose any program
4. Click "Start install"

**Expected behavior**
Programs are installed.

**Additional context**
Using Windows 10 on an Asus laptop

**Comment by tkiral** [comment]
After installing "App Installer" from the windows store, most programs were installed.  LibreWolf was not installed even with this fix.

**Comment by chriswayg** [comment]
I had the same issue because I had debloated Windows 10 with https://github.com/Sycnex/Windows10Debloater first (`.\Windows10SysPrepDebloater.ps1 -Debloat -Privacy`). I also had to reinstall "App Installer" (as @tkiral mentioned). Most apps that I tried (about 20) can now be installed.

For me only _LibreOffice_ did not install.

The script should check for prerequisites such as _App Installer_ and display a meaningful error, rather than displaying a successful install message.
